### PR TITLE
Remove `affiliate` from a few org projects

### PR DESCRIPTION
**Pull request by valencik**
Some of this was originally done in https://github.com/typelevel/typelevel.github.com/pull/424
But I think I messed things up in the big one file squish.
Oh well, I've gone through all the affiliate projects now and I think this is it.

### CODE REVIEW

**Comment by valencik** [comment]
I'm reminded of this annoyance: https://github.com/typelevel/typelevel.github.com/issues/420#issuecomment-1419770016

**Comment by rossabaker** [comment]
Some were flagged as such because they were orginally Incubator projects.  When that went away, nobody had confirmed with the maintainers that they accepted the new terms of the charter, so all Typelevel Organization projects are in the Typelevel GitHub org, but not vice versa.  It would be great if this is reconciled, but that, I think, is why.

**Comment by valencik** [comment]
I was wondering how to reconcile this, and came across the following in the Charter:

> 8.3 Initial projects.** Projects already hosted under the Typelevel GitHub organization when this charter is ratified are considered Organization projects unless they are later removed.

https://github.com/typelevel/governance/blame/227296116f582064bcc33ff63972e06221061647/CHARTER.md#L70

Because we have the option of removing projects in the future, I think this is safe to merge.
Should one of the original maintainers disagree with the Charter rules, the project could be removed.

Pinging the affected teams just as a FYI
@typelevel/kittens @typelevel/simulacrum @typelevel/squants 


I think we should approve this PR and merge to remove any ambiguity in the status of projects.

**Review by jducoeur — APPROVED** [review]

**Review by rossabaker — APPROVED** [review]

**Pull request merged**
